src/openrct2/network/NetworkBase.cpp

```diff
@@ -2568,26 +2566,31 @@ void NetworkBase::Server_Handle_AUTH(NetworkConnection& connection, NetworkPacke
         if (!gameversion || network_get_version() != gameversion)
         {
             connection.AuthStatus = NetworkAuth::BadVersion;
+            log_info("Connection %s: Bad version.", connection.Socket->GetHostName());
```

Comment:
Can we not use `hostName` for the rest of these occurrences of `GetHostName`?

Reply:
I wanted to, must have missed some cases.